docs/medical-api/api-reference/fhir/create-patient-consolidated.mdx

```diff
@@ -24,28 +24,67 @@ api: "POST /medical/v1/patient/{id}/consolidated"
   The entry needs to be an array of [FHIR resources](/medical-api/fhir/resources).
 </ParamField>
 
+<Info>
+  At the moment there is a limit to how much data you can send in a single request. You can send up
+  to 50 resources in a single request and the content-length must not exceed 1Mb.
```

Comment:
nit: `MB` - `b` is bits, `B` is bytes

Reply:
💯 , I copied from another file and didn't notice this. Will update here and there.